examples/legacy/serviceconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "disable_persistent_config": true,
```

Comment:
The service scripts expect to find serviceconfig.json in /examples, so I think that moving this is going to break the service scripts.  For the time being I suggest just leaving this file in place, and not moving to legacy.